privacy-dashboard/privacy-dashboard-impl/src/main/java/com/duckduckgo/privacy/dashboard/impl/ui/PrivacyDashboardHybridViewModel.kt

```diff
@@ -216,7 +212,7 @@ class PrivacyDashboardHybridViewModel @Inject constructor(
         viewModelScope.launch(dispatcher.io()) {
             // when the broken site form is opened from the dashboard, send
             // along a list of params to be sent with the `m_bsr` pixel
-            val siteData = BrokenSiteData.fromSite(site.value, pixelParamList())
+            val siteData = BrokenSiteData.fromSite(site.value)
```

Comment:
Can you also change the fromSite() function? Your previous PR added the params list to the method and this is not needed anymore. That also means removing the boolean_params thing from the BrokenSite report altogether.